src/main/resources/admin/tools/content-type-manager/modules/module-example.html

```diff
@@ -69,7 +67,23 @@
         },
 
         resetJson: (state) => {
-          state.json = emptyJson;
+          const initialJsonElements = [
+            { 
+              type: "element", 
+              name: "content-type", 
+              elements: [
+                { 
+                  type: "element", 
+                  name: "form", 
+                  elements: [] 
+                }
+              ] 
+            } 
+          ];
+          // Used empty array + setTimeout as a quick way to 
+          // remount components and their inner empty states.
+          state.json.elements = [];
+          setTimeout(() => { state.json.elements = initialJsonElements }, 100);
```

Comment:
This seems hacky? Is this the best way to do it? I don't know how mouting works in vue so feel free to ignore